src/main/java/com/denizenscript/depenizen/bukkit/events/projectkorra/EntityBendingDeathScriptEvent.java

```diff
@@ -0,0 +1,79 @@
+package com.denizenscript.depenizen.bukkit.events.projectkorra;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.projectkorra.projectkorra.event.EntityBendingDeathEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class EntityBendingDeathScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // projectkorra player|entity dies|death|killed by <ability>
+    //
+    // @Triggers when an entity dies from bending.
+    //
+    // @Context
+    // <context.ability> returns ElementTag(String) of the ability's name.
+    // <context.source> returns PlayerTag of the player who triggered the ability.
+    // <context.target> returns EntityTag of the target damaged by the ability.
+    // <context.damage> returns ElementTag(Decimal) of the damage dealt to the entity.
+    // <context.element> returns ElementTag(String) ability's element.
+    // <context.is_explosive> returns ElementTag(Boolean) if the ability is explosive.
+    // <context.is_ignite> returns ElementTag(Boolean) if the ability can ignite.
+    // <context.is_sneak> returns ElementTag(Boolean) if the ability is triggered by sneak.
+    //
+    // @Plugin Depenizen, ProjectKorra
+    //
+    // @Player Always.
+    //
+    // -->
+
+    public EntityBendingDeathScriptEvent() {
+        registerCouldMatcher("projectkorra player|entity dies|death|killed by <'ability'>");
+    }
+
+    public EntityBendingDeathEvent event;
+
+    @Override
+    public boolean matches(ScriptPath path) {
+        String ability = path.eventArgLowerAt(4);
+        // Check if event applies to any ability
```

Comment:
Don't need comments that just say what the next line is doing (unless there's something unclear that needs explanation)